pkg/grapicmd/di/providers.go

```diff
@@ -38,8 +38,8 @@ func ProvideUI(ctx *grapicmd.Ctx) clui.UI {
 	return ui
 }
 
-func ProvideCommandExecutor(ctx *grapicmd.Ctx, ui clui.UI) command.Executor {
-	return command.NewExecutor(ctx.OutWriter, ctx.ErrWriter, ctx.InReader)
+func ProvideCommandExecutor(ctx *grapicmd.Ctx, ui clui.UI) excmd.Executor {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideCommandExecutor should have comment or be unexported